src/lib.rs

```diff
@@ -37,6 +37,7 @@ mod events;
 mod reader;
 
 pub use events::MuxedEvents;
+pub use notify::event::*;
```

Comment:
This just comes down to aesthetics/ergonomics, but I feel like some sort of partitioning or namespacing would keep things clean. Maybe `pub use notify::event`? All the types are still accessible, but at least they aren't all exposed at the crate root.

Reply:
Yeah I agree - I think this makes it a bit cleaner.